js/src/pages/price-benchmark/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { getQuery } from '@woocommerce/navigation';
+import { Card } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import MainTabNav from '~/components/main-tab-nav';
+import IssueTypeNavigation from './table-type-navigation';
+import PriceBenchmarkTable from './price-benchmark-table';
+import { TABLE_TYPE_SUGGESTIONS } from './constants';
+
+const PriceBenchmark = () => {
+	const tableType = getQuery()?.tableType || TABLE_TYPE_SUGGESTIONS;
+
+	return (
+		<div className="gla-price-benchmark">
+			<MainTabNav />
+			<Card>
+				<IssueTypeNavigation tableType={ tableType } />
+				<PriceBenchmarkTable tableType={ tableType } />
```

Comment:
@ankitrox I feel we can omit the `PriceBenchmarkTable` here and just display the `PriceBenchmarkSuggestions` and `PriceBenchmarkAdjustments` component directly here.

```suggestion
				{ tableType === TABLE_TYPE_ADJUSTMENTS && (
				<PriceBenchmarkAdjusments />
			) }

			{ tableType === TABLE_TYPE_SUGGESTIONS && (
				<PriceBenchmarkSuggestions />
			) }
```

Alternatively, we can have a map defined as such:
```js
const COMPONENT_TYPE_MAPPING = {
	[ TABLE_TYPE_SUGGESTIONS ]: <PriceBenchmarkSuggestions />,
	[ TABLE_TYPE_ADJUSTMENTS ]: <PriceBenchmarkAdjustments />,
};
```

and then render the appropriate component from the map based on `tableType`:
```js
{ COMPONENT_TYPE_MAPPING[ tableType ] }
```

Reply:
I have used previous suggestion instead of mapping as we only have two components to render conditionally.